_sphinx/extensions/flutter_app.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+/// Create an overlay with an iframe, the iframe's source is [url]. This also
+/// creates an (x) button to hide the overlay.
+function run_flutter_app(url) {
+  let id = compute_iframe_id(url);
+  create_overlay();
+  if (!$('#' + id).length) {
+    $('#flutter-app-overlay').append($(
```

Comment:
jQuery! this reminds me of simpler times...

Reply:
Simpler times when everything was garbage, but it was very simple to build your house from that garbage, you just piled it on top of each other and crawled in. 😄